stable/mission-control/templates/elasticsearch-secrets.yaml

```diff
@@ -9,4 +10,13 @@ metadata:
     release: {{ .Release.Name }}
 type: Opaque
 data:
-  password:  {{ required "A valid .Values.elasticsearch.password entry required!" .Values.elasticsearch.password | b64enc | quote }}
+{{- if .Values.elasticsearch.username }}
+  username:  {{ .Values.elasticsearch.username | b64enc | quote }}
```

Comment:
What happens when there's no username and password?

Reply:
@danielezer  we have default user/password  set in values.yaml